zerver/context_processors.py

```diff
@@ -86,6 +86,8 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
     if settings.ZILENCER_ENABLED:
         apps_page_url = '/apps/'
 
+    apps_page_web = settings.ROOT_DOMAIN_URI+'/accounts/go/'
```

Comment:
Please spell this with space around the operators ( ` + `)

Reply:
Is it okay now?